specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```
@@ -0,0 +1,1192 @@
+              }
+            ]
+          }
+        }
+      },
+      "type": "object"
+    },
+    "AnomalyInterpretation": {
+      "properties": {
+        "variable": {
+          "type": "string",
+          "description": "Variable.",
+          "example": "variable_1"
+        },
+        "contributionScore": {
+          "type": "number",
+          "description": "This score shows the percentage contributing to the anomalous timestamp. A number between 0 and 1.",
+          "example": [
+            0.3324159383
+          ]
+        },
+        "correlationChanges": {
+          "example": {
+            "changedVariables": [
+              "variable_2",
+              "variable_3"
+            ]
+          },
+          "$ref": "#/definitions/CorrelationChanges"
+        }
+      },
+      "type": "object"
+    },
+    "CorrelationChanges": {
+      "properties": {
+        "changedVariables": {
+          "type": "array",
+          "description": "The correlated variables that have correlation changes under an anomaly.",
+          "example": [
+            "variable_2",
+            "variable_3"
+          ],
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "LastDetectionRequest": {
+      "required": [
+        "topContributorCount",
+        "variables"
+      ],
+      "properties": {
+        "variables": {
+          "type": "array",
+          "description": "Variables",
+          "example": [
+            {
+              "variableName": "Variable_1",
+              "timestamps": [
+                "2022-01-01T00:00:00Z",
+                "2022-01-01T00:01:00Z",
+                "2022-01-01T00:02:00Z"
+              ],
+              "values": [
+                0.4551378545933972,
+                0.7388603950488748,
+                0.201088255984052
+              ]
+            }
+          ],
+          "items": {
+            "$ref": "#/definitions/VariableValues"
+          }
+        },
+        "topContributorCount": {
+          "type": "integer",
+          "description": "An optional field, which is used to specify the number of top contributed variables for one anomalous timestamp in the response. The default number is 10.",
+          "example": 10
+        }
+      },
+      "type": "object"
+    },
+    "VariableValues": {
+      "required": [
+        "name",
+        "timestamps",
+        "values"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Variable name.",
+          "example": "variable_1"
+        },
+        "timestamps": {
+          "type": "array",
+          "description": "Timestamp",
+          "example": "2022-04-02T00:00:00Z",
+          "items": {
+            "type": "string"
+          }
+        },
+        "values": {
+          "type": "array",
+          "description": "Values of variables.",
+          "example": "1.2",
```

`values` is `type: array` with `items: { type: number }` so its example must be an array of numbers.
```suggestion
          "example": [ 1.2 ],
```